content/blog/2019-08-15-new-react-devtools.md

```diff
@@ -37,14 +37,14 @@ If you use the standalone shell (e.g. in React Native or Safari), you can instal
 npm install -g react-devtools@^4
 ```
 
-## Where did all of the DOM elements go?
+## Where did all of the DOM elements go? {#where-did-all-of-the-dom-elements-go}
```

Comment:
same as above

`## Where did all of the DOM elements go? {#where-did-all-of-the-dom-elements-go}` => `## Where did all of the DOM elements go?`